pkg/index/validation/validate_test.go

```diff
@@ -190,8 +190,8 @@ func TestValidatePlatform(t *testing.T) {
 			wantErr:  true,
 		},
 		{
-			name:     "no file operations",
-			platform: testutil.NewPlatform().WithFiles(nil).V(),
+			name:     "empty file operations",
```

Comment:
can you also add WithFiles(nil) case (`wantErr:false`)?

Reply:
Those test cases are covered by `Test_validateFiles`. I only included this error case in this test, to check if files are validated at all. Do you think this is not enough?